header-only/layout-builder/awkward/utils.h

```diff
@@ -270,6 +271,47 @@ namespace awkward {
     visit_impl<sizeof...(CONTENTs)>::visit(contents, index, fun);
   }
 
+  /// @brief Helper function to retrieve the names of the buffers.
+  ///
+  /// Only to to used with C interface. Not recommended as it can cause a mismatch
```

Comment:
I'd say: "Note: use with caution, beware of a potential mismatch between retrieved values!"